scripts/release-ci.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Do a complete release. Run on CI.
+# Upload assets, run goreleaser, and notify Tilt Cloud of the new release binaries.
+
+set -ex
+
+if [[ "$GITHUB_TOKEN" == "" ]]; then
+    echo "Missing GITHUB_TOKEN"
+    exit 1
+fi
+
+if [[ "$DOCKER_TOKEN" == "" ]]; then
+    echo "Missing DOCKER_TOKEN"
+    exit 1
+fi
+
+if [[ "$TILT_CLOUD_TOKEN" == "" ]]; then
+    echo "Missing Tilt release token"
+    exit 1
+fi
+
+DIR=$(dirname "$0")
+cd "$DIR/.."
+
+echo "$DOCKER_TOKEN" | docker login --username "$DOCKER_USERNAME" --password-stdin
+
+mkdir -p ~/.tilt-dev
+echo "$TILT_CLOUD_TOKEN" > ~/.tilt-dev/token
+
+git fetch --tags
+./scripts/upload-assets.py latest
+goreleaser --rm-dist --skip-publish --snapshot
+
+VERSION=$(git describe --abbrev=0 --tags)
```

Comment:
If you want, I think `$CIRCLE_TAG` instead of `latest` on 32 and `$VERSION` elsewhere would be a little more direct and leave less room for things to get weird (like, if a config error/change causes this job to run on a branch instead of a tag, the current code would use the latest tag rather than failing), but I don't think it's a big deal